lighthouse-core/audits/byte-efficiency/unminified-css.js

```diff
@@ -53,15 +53,14 @@ class UnminifiedCSS extends ByteEfficiencyAudit {
   /**
    * @param {LH.Artifacts.CSSStyleSheetInfo} stylesheet
    * @param {LH.Artifacts.NetworkRequest|undefined} networkRecord
-   * @param {string} pageUrl
    * @return {{url: string, totalBytes: number, wastedBytes: number, wastedPercent: number}}
    */
-  static computeWaste(stylesheet, networkRecord, pageUrl) {
+  static computeWaste(stylesheet, networkRecord) {
     const content = stylesheet.content;
     const totalTokenLength = UnminifiedCSS.computeTokenLength(content);
 
     let url = stylesheet.header.sourceURL;
-    if (!url || url === pageUrl) {
+    if (!url || stylesheet.header.isInline) {
       const contentPreview = UnusedCSS.determineContentPreview(stylesheet.content);
       url = contentPreview;
```

Comment:
(unrelated to pr) ugh we shouldn't be putting non-url strings in url fields. third party filter wont like this